### PR TITLE
added AWS healthlake datastore

### PR DESCRIPTION
**Pull request by wrschneider**

### CODE REVIEW

File: resources/healthlake-datastore.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        params := &healthlake.ListFHIRDatastoresInput{}
+
+        for {
+                output, err := svc.ListFHIRDatastores(params)
+                if err != nil {
+                        return nil, err
+                }
+
+                for _, datastore := range output.DatastorePropertiesList {
+                        if *datastore.DatastoreStatus != "DELETED" {
```
Comment (shubydo) [inline resources/healthlake-datastore.go:30]:
```suggestion
                        if *datastore.DatastoreStatus != healthlake.DatastoreStatusDeleted {
```
Another option here is to use the constant defined in the package [here](https://pkg.go.dev/github.com/aws/aws-sdk-go@v1.44.7/service/healthlake#pkg-constants)

**Review by shubydo — COMMENTED** [review]
Minor comment. Were you able to test this using your PR [here](https://github.com/Optum/dce/pull/414)?

Side note: Don't believe this is part of the upstream repo yet, but it's probably a good opportunity to get our fork updated and contribute this back after it's merged. This would also close #9

**Comment by wrschneider** [comment]
This PR may be moot now.

At some point the main aws-nuke repo added support for AWS Cloud Control API which can generically list/delete many resources without having explicit resource-specific Go code for each.

**Comment by bjfish25** [comment]
Closing this to clean up prs. Be free to re-open if want merged

**Pull request closed without merging**